config.json

```diff
@@ -1143,6 +1143,62 @@
     ]
   },
   "concepts": [],
-  "key_features": [],
-  "tags": []
+  "key_features": [
+    {
+      "title": "Evolving",
+      "content": "Typescript is a modern and constantly evolving language supported by Microsoft."
+    },
+    {
+      "icon": "features-cross-platform",
+      "title": "Widely used",
+      "content": "Typescript is used by many frontend frameworks, and can be used in Javascript based ones as well."
+    },
+    {
+      "icon": "features-strongly-typed",
+      "title": "Strongly typed",
+      "content": "Typescript flexible typing system lets you describe what to expect of your code easily."
+    },
+    {
+      "icon": "features-multi-paradigm",
+      "title": "Multi-paradigm",
+      "content": "Mainly focused in OOP, it also has tools to support other paradigms like Functional Programming."
+    },
+    {
+      "icon": "features-packages",
+      "title": "Library and tooling",
+      "content": "Typescript is able to use all of Javascript vast choice of library and tooling, plus its own."
```

Comment:
```suggestion
      "content": "TypeScript's integration is handled in the language itself, enabling its features in many editors and tools."
```

Reply:
Eh. I'm not to fond of my suggested change, but 🤷🏽 . See if you like it.

Reply:
I added JavaScript to make it more clear `TypeScript's JavaScript integration is handled by the language itself, enabling the use of its tools and features`  I don't know how to put the `editor` part tho without it sounding strange >.< I'll think about that

Reply:
Shortened it to `TS's JavaScript integration is handled by the language itself, enabling its tools and features` because of the max size in the linter

Reply:
Fine for now, we can always change it later 🗡️